### ISSUE TITLE
Insert of second image corrupts sixel graphics

### ISSUE THREAD

**Issue by Yaraslaut** [contour-terminal/contour#1049]
### Contour Terminal version

Contour Terminal Emulator 0.3.11-master-cd0aa937

### Installer source

Github: source code cloned

### Operating System

RHEL

### Architecture

x86-64

### Other Software

julia

### Steps to reproduce

https://user-images.githubusercontent.com/44506630/221343690-788f3798-f458-4d73-8e12-e5bebfc2bc9a.mp4

### Actual Behavior

sixel rendering is broken

### Additional notes

To reproduce this behavior size of the terminal in pixels must exceed 1080p

[figure](https://user-images.githubusercontent.com/44506630/221111967-293fd7e5-a82f-485d-bbfa-a7e6407b92fd.png)
[lines](https://user-images.githubusercontent.com/44506630/221111970-1fddf1a7-48b8-40d4-b351-7f9a20e29623.png)

**Comment by christianparpart** [comment]
I am looking at this from my phone, sorry for this. Is the problem that at the end of the scroll, the background image is shining through?

**Comment by christianparpart** [comment]
I wonder if this is only happening on qml branch or also on master. 🤔

**Comment by Yaraslaut** [comment]
> I wonder if this is only happening on qml branch or also on master. thinking

No, I am seeing same behaviour on current master as well `Contour Terminal Emulator 0.3.11-master-99d6e65a`

**Comment by christianparpart** [comment]
Ok thx. This was just a hunch.

**Comment by Yaraslaut** [comment]
> I am looking at this from my phone, sorry for this. Is the problem that at the end of the scroll, the background image is shining through?

The problem is that after i insert second image first becomes corrupted and i see this:
![image](https://user-images.githubusercontent.com/44506630/221489254-7aeef5af-a22a-4dc1-b45c-6b639b37f656.png)

**Comment by Yaraslaut** [comment]
Can be reproduced with one image, example julia code 
`f = Figure(resolution=(2000,1100)); ax = Axis(f[1,1]); lines!(ax,rand(100)); f`

**Comment by christianparpart** [comment]
We figured out that #1458 is only a interim workaround but not an ultimate fix.

What should be done to close this ticket is to probably have a atlas texture size with the dimension of the current screen this window is living in.

- if the window is moved to another screen (or the physical screen size is changed), then the texture atlas needs to be re-allocated
- if the font size is changed only, then the texture atlas itself only needs to be re-addressed and re-initialized but not re-created.
